engine/src/main/java/org/camunda/bpm/engine/impl/jobexecutor/FailedJobListener.java

```diff
@@ -48,6 +48,11 @@ public Void execute(CommandContext commandContext) {
 
     logJobFailure(commandContext);
 
+    if (isJobReacquired(commandContext)) {
```

Comment:
Can we log that the failed job command has been skipped? Like this, it is easier to root cause issues that users might face in the future.